### PR TITLE
Support Swift 4.0

### PR DESCRIPTION
**Pull request by dduan**
Change required is minimal.

1. Declare Swift 4 Support.
2. Update syntax for 1 test.

### CODE REVIEW

File: Tests/KronosTests/ClockTests.swift [hunk 1]
```diff
@@ -33,8 +33,8 @@ final class ClockTests: XCTestCase {
         let firstExpectation = self.expectation(description: "Clock sync calls first closure")
         let lastExpectation = self.expectation(description: "Clock sync calls last closure")
         Clock.sync(
-            first: { _ in lastExpectation.fulfill() },
-            completion: { _ in firstExpectation.fulfill() })
+            first: { _, _ in lastExpectation.fulfill() },
```
Comment (keith) [inline Tests/KronosTests/ClockTests.swift]: can remove this change now

Comment (dduan) [reply Tests/KronosTests/ClockTests.swift]: no, unfortunately.

Comment (dduan) [reply Tests/KronosTests/ClockTests.swift]: screen shot 2017-08-28 at 2 19 10 pm

**Comment by Reflejo** [comment]
@dduan do the test pass from your machine?

**Comment by Reflejo** [comment]
@dduan check my question above, if it does just merge it

**Pull request merged**